src/components/panels/hours-lite-panel.js

```diff
@@ -1,16 +1,16 @@
 import React, { useState, useEffect } from 'react';
 import { SPACING, Icon, TYPOGRAPHY, COLORS, Heading } from '../../reusable';
-import * as moment from 'moment';
+import dayjs from 'dayjs';
 import { Link as GatsbyLink } from 'gatsby';
 
 import Link from '../link';
 import icons from '../../reusable/icons';
 import MEDIA_QUERIES from '../../reusable/media-queries.js';
 import { displayHours } from '../../utils/hours';
 
-export default function HoursLitePanel({ data }) {
+export default function HoursLitePanel ({ data }) {
   const [initialized, setInitialized] = useState(false);
-  const { field_title } = data;
+  const { field_title: fieldTitle } = data;
```

Comment:
Clever approach!

Reply:
Thanks! It feels a bit hacky, though. I personally like differentiating between JS variables / imported data variables with snake case and camel case. Have you encountered this yourself? If so, did you have another solution that I could implement for consistency?